Framework/Helpers/RandomCharsGenerators.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using OpenQA.Selenium;
+
+namespace Framework.Helpers
+{
+    public static class RandomCharsGenerators
+    {
+        public static string CreateRandomChars(int textLength)
```

Comment:
You can use https://metanit.com/sharp/tutorial/15.1.php

public static string RandomString(int length)
{
    const string chars = "abcdefghijkmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789!@$?_-";
    Random rd = new Random();
    return new string(Enumerable.Repeat(chars, length)
      .Select(s => s[rd.Next(s.Length)]).ToArray());
}

Reply:
I have used your code.